about.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Kaitlin Forsman :: About</title>
+    <link href="https://fonts.googleapis.com/css?family=Fredericka+the+Great|Gentium+Book+Basic" rel="stylesheet">
+    <link rel="stylesheet" href="styles/normalize.css">
+    <link rel="stylesheet" href="styles/style.css">
+    <link rel="stylesheet" href="styles/about.css">
+  </head>
+  <body>
+    <div class="container-all">
```

Comment:
I suggest making this a `section` tag or just use the `body` as the container for all instead of a `div`.